src/main/java/io/voucherify/client/module/VoucherModule.java

```diff
@@ -254,5 +254,17 @@ public ImportVouchersResponse method() {
             }
           });
     }
+
+    public Observable<Irrelevant> releaseSession(final String code, final String sessionKey) {
```

Comment:
please also add this method under `ExtAsync`, as right now it is not available when using callback approach

Reply:
Done, I also added `delete` which was missing